user_tools/src/spark_rapids_tools/tools/qualx/qualx_main.py

```diff
@@ -506,7 +506,10 @@ def train(
     ensure_directory(output_dir)
 
     for split in ['train', 'test']:
-        features_split = features[feature_cols].loc[features['split'] == split]
+        if split == 'train':
```

Comment:
Do the below filenames also need to indicate that we are using `train + val` features?

Reply:
No, we only save the train split metrics with the model per L523, otherwise the feature importance shap files have a split suffix on L522.

Reply:
And we actually train the model on both train and val, but just split up for local eval during training steps.

Reply:
I see. Thanks @leewyang.